src/object/object_manager.cpp

```diff
@@ -141,7 +141,9 @@ CObject* CObjectManager::CreateObject(ObjectCreateParams params)
             m_nextId = params.id + 1;
         }
     }
-
+    
```

Comment:
Linter complains about whitespace here too but I don't see anything huh. Maybe \r\n instead of \n?